### ISSUE TITLE
[UX] Improvement needed for Category Selection to avoid confusion

### ISSUE THREAD

**Issue by leowmjw** [mysociety/fixmystreet#1323]
Observed some Usability issues detailed below in Malaysia's FMS implementation aduanku.my; would like to offer some UX suggestion when choosing the Category Selection.

As an example; see two issues filed in the same Administrative Area (PPR Kota Damansara); where it is in the larger Administrative Area of "MBPJ Zon 3"
```
Incorrect: https://aduanku.my/report/837
Correct: https://aduanku.my/report/836
```
What would be useful; consider the following 3 implementation:
```
a) Append a text of the Body name before the Category name (e.g. PPR Kota Damansara/Maintenance, MBPJ Zon 3/Kerbersihan ); with the most specific Body (in the hierarchy) ordered first

b) Allow the reporter to specifically select the Body (in the case below, "PPR Kota Damansara" instead of "MBPJ Zon 3") before selecting the Category; with the most specific Body (in the hierarchy) selected by default

c) Group Categories together by Bodies (have a separator); with the most specific Body (in the hierarchy) ordered first
```

**Comment by dracos** [comment]
Hi – thanks for the info. As I'm sure you're aware, the FixMyStreet code directs reports according to the category selected, so different categories can be sent to different administrative bodies, depending on who is responsible for that category of issue. We didn't really want to include the body within the category, because a user of the site shouldn't need to know anything about the body responsible for the issue, they should just pick the type of report and it is sent accordingly.

Of course, this might not work everywhere, but I'm not totally clear from your description (I'm not sure what the category names mean, sorry) why you want to include the body. I assume you have some categories for which both bodies, PPR Kota and MBPJ Zon 3, are responsible in different ways? Say they both deal with potholes, you could either have a Potholes category in both bodies, and then reports in that category would be sent to both; or if you're happy your users would know which body to pick, you could have a "Potholes (PPR Kota)" category in the PPR Kota body, and a "Potholes (MBPJ Zon 3)" category in MBPJ Zon 3 body, and then both Potholes categories would appear in the front end with the body name and could be picked accordingly.

If you had the body name first, then people would have to know which body was responsible for what before being able to find the correct category, which I would find confusing. If I've misunderstood, please do provide some more details; but I would encourage you not to require your users to know which body to select if that's not something that they might immediately know.

**Comment by leowmjw** [comment]
Ok, so what you are saying is if it has the same category name, it will be sent to both bodies, but in the drop down it will only show one time. Is my understanding correct?

So in the case of pothole, the drop down list shows only one time and thus the user do not need to know which authority to report under.

I understand now the rationale, definitely think it is worth to document examples like this is the doc, would have been useful when I was reading through it.

Thanks! I'll try it out

Sent from Outlook Mobile<https://aka.ms/blhgte>



On Tue, Feb 2, 2016 at 3:15 AM -0800, "M Somerville" <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi - thanks for the info. As I'm sure you're aware, the FixMyStreet code directs reports according to the category selected, so different categories can be sent to different administrative bodies, depending on who is responsible for that category of issue. We didn't really want to include the body within the category, because a user of the site shouldn't need to know anything about the body responsible for the issue, they should just pick the type of report and it is sent accordingly.

Of course, this might not work everywhere, but I'm not totally clear from your description (I'm not sure what the category names mean, sorry) why you want to include the body. I assume you have some categories for which both bodies, PPR Kota and MBPJ Zon 3, are responsible in different ways? Say they both deal with potholes, you could either have a Potholes category in both bodies, and then reports in that category would be sent to both; or if you're happy your users would know which body to pick, you could have a "Potholes (PPR Kota)" category in the PPR Kota body, and a "Potholes (MBPJ Zon 3)" category in MBPJ Zon 3 body, and then both Potholes categories would appear in the front end with the body name and could be picked accordingly.

If you had the body name first, then people would have to know which body was responsible for what before being able to find the correct category, which I would find confusing. If I've misunderstood, please do provide some more details; but I would encourage you not to require your users to know which body to select if that's not something that they might immediately know.

-

**Comment by dracos** [comment]
Yes, that's correct :) Sorry that it's not documented, I'll look at getting something added as soon as possible.

**Comment by leowmjw** [comment]
@dracos It works!  If you can suggest where in the repo I should document this; I can contribute a small write-up and/or screenshots.

**Comment by dracos** [comment]
Great, and thanks :) I guess somewhere on http://fixmystreet.org/running/bodies_and_contacts/ would be the best page, the docs are in the gh-pages branch of this repository.
